packages/block-editor/src/hooks/custom-fields.js

```diff
@@ -55,7 +56,11 @@ const withInspectorControl = createHigherOrderComponent( ( BlockEdit ) => {
 
 		// Check if the current block is a paragraph or image block.
 		// Currently, only these two blocks are supported.
-		if ( ! [ 'core/paragraph', 'core/image' ].includes( props.name ) ) {
+		if (
+			! [ 'core/paragraph', 'core/image', 'core/heading' ].includes(
```

Comment:
We discussed enabling the Button block, too. It's fine to take care of it as a follow-up. I'm leaving it mostly as a note.